app/services/tock_importer.rb

```diff
@@ -14,10 +14,6 @@ def perform
   private
 
   def projects
-    @_projects ||= json["results"]
```

Comment:
the API changed. If you look at the updates to the fixtures, the API now returns an array rather than a JSON object with a `results` key